opentelemetry-stdout/src/common.rs

```diff
@@ -0,0 +1,150 @@
+use std::{borrow::Cow, collections::BTreeMap};
+
+use serde::Serialize;
+
+#[derive(Debug, Serialize, Clone)]
+pub(crate) struct AttributeSet(BTreeMap<Key, Value>);
+
+impl From<&opentelemetry_sdk::AttributeSet> for AttributeSet {
+    fn from(value: &opentelemetry_sdk::AttributeSet) -> Self {
+        AttributeSet(
+            value
+                .iter()
+                .map(|(key, value)| (Key::from(key.clone()), Value::from(value.clone())))
+                .collect(),
+        )
+    }
+}
+
+#[derive(Debug, Clone, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct Resource {
+    attributes: Vec<KeyValue>,
+    #[serde(skip_serializing_if = "is_zero")]
+    dropped_attributes_count: u64,
```

Comment:
Does resource need attribute_count limits?
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/README.md#exempt-entities

Reply:
It probably should get one in a follow up, the otlp spec includes this as well so we should probably record it.